### DISCUSSION TITLE
VLC 4.x support

### DISCUSSION THREAD

**Discussion by Bestfast** [Syncplay/syncplay#576] [Ideas]
Since VLC 4.0 is in a state where it's usable and nightly builds are being released everyday, is there any plan on supporting it?

**Comment by Et0h** [comment]
I just had a quick try with https://raw.githubusercontent.com/Syncplay/syncplay/VLC4/syncplay/resources/lua/intf/syncplay.lua and it didn't work. I don't know if the problem is with the script, with Syncplay or with VLC4.

If someone wants to try to get it to work then feel free, but I don't use VLC 4 so I'm probably not the best person to introduce and maintain support for it.

**Comment by Bestfast** [comment]
I see, thanks for the answer. I'm not a lua programmer myself but just another Syncplay user. 

I can see why it might be hard to maintain support and implement it for you, but I'd still like to thank you for your attempt on doing so. I'd be glad to be a tester if anybody else is going to try it, but sadly I cannot do more.
